samples/dotnet/ConsoleConsumer/LocalSchemaRegistry.cs

```diff
@@ -25,15 +25,12 @@ public int MaxCachedSchemas
             }
```

Comment:
Wonder if we still need this file to be honest.

Reply:
Removed!